packages/catalog-search/src/SearchBox.jsx

```diff
@@ -66,9 +69,9 @@ export const SearchBoxBase = ({
         onSubmit={handleSubmit}
         onClear={handleClear}
       >
-        <SearchField.Input className="form-control-lg" aria-labelledby="search-input-box" data-hj-whitelist />
-        <SearchField.ClearButton />
-        <SearchField.SubmitButton />
+        <SearchField.Input className="form-control-lg" aria-labelledby="search-input-box" data-hj-whitelist data-nr-synth-id="input-field" />
+        <SearchField.ClearButton data-nr-synth-id="clear-button" />
+        <SearchField.SubmitButton data-nr-synth-id="submit-button" />
```

Comment:
Curious if the attribute values here should be a bit more specific to Catalog Search? I'm thinking there could potentially be some other submit button or input field on the page outside of the catalog-search's `SearchBox` component and if it also had generic "submit-button" or "input-field", would NewRelic be able to distinguish between these?

Reply:
Hmm, good point. I believe there's a chance for that situation, and while theoretically would be possible to distinguish within New Relic, it would add a bunch more complexity needed within the Synthetics test which could easily be solved by simply adding a `catalog-search` to the beginning of the ID's here- so I'll add